zb-classifier.js

```diff
@@ -531,14 +552,17 @@ class ZigbeeClassifier {
     return property;
   }
 
-  addButtonMotionProperty(node, genOnOffOutputEndpoint) {
+  addButtonMotionProperty(node, genOnOffOutputEndpoint, suffix) {
+    if (typeof suffix === 'undefined') {
+      suffix = '';
+    }
     const property = this.addProperty(
       node,                           // device
-      'motion',                       // name
+      `motion${suffix}`,                       // name
```

Comment:
nit: comment spacing should align with previous line

Reply:
done